erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -133,7 +133,7 @@ frappe.ui.form.on("BOM", {
 	make_work_order: function(frm) {
 		const fields = [{
 			fieldtype: 'Float',
-			label: __('Qty To Manufacture'),
+			label: frm.doc.manufacturing_type == "Process" ? __('Raw Material Qty') : __('Qty To Manufacture'),
 			fieldname: 'qty',
 			reqd: 1,
 			default: 1
```

Comment:
Default this to the set quantity, based on the manufacturing type:

```suggestion
			default: frm.doc.manufacturing_type == "Process" ? frm.doc.items[0].qty : frm.doc.quantity
```